docs/user-guide/kubectl/kubectl_expose.md

```diff
@@ -69,10 +69,11 @@ $ kubectl expose rc streamer --port=4100 --protocol=udp --name=video-stream
       --dry-run=false: If true, only print the object that would be sent, without creating it.
       --generator="service/v2": The name of the API generator to use. There are 2 generators: 'service/v1' and 'service/v2'. The only difference between them is that service port in v1 is named 'default', while it is left unnamed in v2. Default is 'service/v2'.
   -h, --help=false: help for expose
+  -j, --jsonpath="": JSONPath template string to use when -o=jsonpath. The jsonpath template is composed of jsonpath expressions enclosed by {}
```

Comment:
that's fine with me too.

@daizuozhuo do you want to do a separate PR to do that?